packages/atclient/include/atclient/metadata.h

```diff
@@ -4,48 +4,49 @@
 
 #include "atclient/atsign.h"
 #include "atclient/atstr.h"
+#include "cJSON/cJSON.h"
 #include <stdbool.h>
 #include <stddef.h>
-#include <stdint.h>
+#include <stdint.h> // IWYU pragma: keep (for uint8_t)
```

Comment:
My LSP was saying this was unused, but it isn't hence the added pragma: keep so there aren't any warnings. This has to do with the use of aggressive clangd settings to make sure we don't have unused includes in the code. (IWYU means "include what you use")